frontend/packages/patternfly/locales/en/notification-drawer.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Notifications": "Notifications",
```

Comment:
Hi @zherman0, what is creating this file in `packages/patternfly`?  It looks like you removed all the `~notification-drawer` i18n namespaces.  How is it different from the deleted `frontend/public/locales/en/notification-drawer.json`?

Reply:
@dtaylor113  -  package specific translations need to be in the package.